### PR TITLE
Docsite: add Community guide

### PR DESCRIPTION
**Pull request by Andersson007**
Dear maintainers,
As a part of the [Consolidating Ansible discussion platforms initiative](https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812), this PR adds the communication section template defined by the community to the README and the docsite. Similar PRs are now being raised across all the Ansible ecosystem projects.

Thank you

### CODE REVIEW

**Comment by Andersson007** [comment]
ready for review

File: docs/community.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Community
+
+We welcome your feedback, questions and ideas. Here's how to reach the community.
+
+## Forum
+
+Join the [Ansible Forum](https://forum.ansible.com) as a single starting point and our default communication platform for questions and help, development discussions, events, and much more. [Register](https://forum.ansible.com/signup?) to join the community. Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!
+
+* [Get Help](https://forum.ansible.com/c/help/6): get help or help others. Please add appropriate tags if you start new discussions, for example `antsibull`.
```
Comment (felixfontein) [inline docs/community.md:9]: I don't think Get Help makes sense here. Collection Development is way better suited as users of antsibull-changelog will almost always be collection developers.

File: docs/community.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+* [Posts tagged with 'antsibull'](https://forum.ansible.com/tag/antsibull): subscribe to participate in project-related conversations.
+* [Social Spaces](https://forum.ansible.com/c/chat/4): gather and interact with fellow enthusiasts.
+* [News & Announcements](https://forum.ansible.com/c/news/5): track project-wide announcements including social events. The [Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn), which is used to announce releases and important changes, can also be found here.
+
+For more information on the forum navigation, see [Navigating the Ansible forum](https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39) post.
```
Comment (felixfontein) [inline docs/community.md:14]: There should be a second section in this document that references the Antsibull Matrix room (`#antsibull:ansible.com`).

File: docs/index.md [hunk 3]
```diff
@@ -10,6 +10,8 @@ SPDX-License-Identifier: GPL-3.0-or-later
 [![Nox badge](https://github.com/ansible-community/antsibull-changelog/workflows/nox/badge.svg?event=push&branch=main)](https://github.com/ansible-community/antsibull-changelog/actions?query=workflow%3A%22nox%22+branch%3Amain)
 [![Codecov badge](https://img.shields.io/codecov/c/github/ansible-community/antsibull-changelog)](https://codecov.io/gh/ansible-community/antsibull-changelog)
 
+> Need help or want to discuss the project? See our [Community guide](https://ansible.readthedocs.io/projects/antsibull-changelog/community/) to learn how to join the conversation!
```
Comment (felixfontein) [inline docs/index.md:13]: I don't think this should come **before** the main information for the project itself. Can you please move it before line 27 (i.e. before the start of the next section)?

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

File: docs/community.md
Comment (felixfontein) [inline docs/community.md:1]: This document is missing a copyright / license statement. Check out the other MD files.

File: docs/index.md (hunk 3 above)
Comment (felixfontein) [inline docs/index.md:13]:
```suggestion
> Need help or want to discuss the project? See our [Community guide](community.md) to learn how to join the conversation!
```

**Review by felixfontein — COMMENTED** [review]
You also need to mention `documentation.md` in `mkdocs.yml`'s `nav` list.

File: docs/community.md (hunk 1 above)
Comment (Andersson007) [reply docs/community.md]: we should rename it to "Give Help" here:) anyway, SGTM

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing! All done, PTAL

File: docs/community.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+# Community
+
+We welcome your feedback, questions and ideas. Here's how to reach the community.
+
+## Forum
+
+Join the [Ansible Forum](https://forum.ansible.com) as a single starting point and our default communication platform for questions and help, development discussions, events, and much more. [Register](https://forum.ansible.com/signup?) to join the community. Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!
```
Comment (felixfontein) [inline docs/community.md:13]: One-word link titles should be avoided (for accessibility reasons)..
```suggestion
Join the [Ansible Forum](https://forum.ansible.com) as a single starting point and our default communication platform for questions and help, development discussions, events, and much more. [Register on the Forum](https://forum.ansible.com/signup?) to join the community. Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!
```

File: docs/community.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+* [Posts tagged with 'antsibull'](https://forum.ansible.com/tag/antsibull): subscribe to participate in project-related conversations.
+* [Social Spaces](https://forum.ansible.com/c/chat/4): gather and interact with fellow enthusiasts.
+* [News & Announcements](https://forum.ansible.com/c/news/5): track project-wide announcements including social events. The [Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn), which is used to announce releases and important changes, can also be found here.
+
+For more information on the forum navigation, see the [Navigating the Ansible forum](https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39) post.
+
+## Matrix
+
+For real-time interactions, join the [#antsibull:ansible.com](https://matrix.to/#/#antsibull:ansible.com) Matrix channel. See the [Ansible communication guide](https://docs.ansible.com/ansible/devel/community/communication.html#real-time-chat) for more information.
```
Comment (felixfontein) [inline docs/community.md:23]:
```suggestion
For real-time interactions, join the [#antsibull:ansible.com Matrix room](https://matrix.to/#/#antsibull:ansible.com). See the [Ansible communication guide](https://docs.ansible.com/ansible/devel/community/communication.html#real-time-chat) for more information on Matrix.
```

**Comment by Andersson007** [comment]
@felixfontein committed, lgtm, thanks, PTAL

**Review by samccann — APPROVED** [review]
LGTM

File: docs/index.md [hunk 6]
```diff
@@ -22,6 +22,8 @@ antsibull-changelog is covered by the [Ansible Code of Conduct](https://docs.ans
 To see how antsibull-changelog changes, please look at the
 [antsibull-changelog changelog](https://github.com/ansible-community/antsibull-changelog/blob/main/CHANGELOG.md).
 
+> Need help or want to discuss the project? See our [Community guide](community.md) to learn how to join the conversation!
+
```
Comment (samccann) [inline docs/index.md:26]: Is this really how you want this to look?
image

If not, remove the `>` above.

**Review by samccann — COMMENTED** [review]
one minor nit

File: docs/index.md (hunk 6 above)
Comment (felixfontein) [reply docs/index.md:26]: Alternatively, you can make it a proper note:
```md
!!! note
    Need help or want to discuss the project? See our [Community guide](community.md) to learn how to join the conversation!
```

File: docs/index.md [hunk 7]
```diff
@@ -22,6 +22,8 @@ antsibull-changelog is covered by the [Ansible Code of Conduct](https://docs.ans
 To see how antsibull-changelog changes, please look at the
 [antsibull-changelog changelog](https://github.com/ansible-community/antsibull-changelog/blob/main/CHANGELOG.md).
 
+> Need help or want to discuss the project? See our [Community guide](community.md) to learn how to join the conversation!
```
Comment (felixfontein) [inline docs/index.md:25]:
```suggestion
!!! note
    Need help or want to discuss the project? See our [Community guide](community.md) to learn how to join the conversation!
```

File: docs/index.md (hunk 6 above)
Comment (felixfontein) [reply docs/index.md:27]: The result looks like this:
image

**Review by felixfontein — APPROVED** [review]

File: docs/index.md (hunk 6 above)
Comment (Andersson007) [reply docs/index.md:27]: LGTM, thanks! didn't know that

**Comment by Andersson007** [comment]
@felixfontein @samccann thanks for reviewing!

**Comment by felixfontein** [comment]
@Andersson007 thanks for implementing this (and the other PR)!

**Pull request merged**